src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Alert coverage gaps
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+  - Alert coverage gaps
+metaDescription: Learn how alert coverage gaps in New Relic help you identify gaps in your alert system. 
+---
+import alertcoveragegaps from 'images/alert-coverage-gaps-homepage.png'
+
+import addanalert from 'images/add-an-alert.png'
+
+import createapolicyalertcoveragegaps from 'images/create-a-policy-coverage-gaps.png'
+
+Alert Coverage gaps highlight areas in your stack where you're missing observability coverage. In our Alerts and AI section [we've created a homepage](https://one.newrelic.com/nrai/detection-gaps/home) where you can see all your uncovered entities so you can assess what information you're team is lacking and address the knowledge gap. 
+
+In an increasingly dynamic landscape, we know how important it is to stay on top of your stack and be proactively notified if anything goes wrong. But, as your team grows or changes, it can be hard to know if you're getting the alerts you need, when you need them. Alert coverage gaps gives your team a comprehensive list of all entities that your team *is not currenty covering*.  
+
+What does covering an entity mean? A covered entity means that you've set up an [alert](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/) to notify you of how a particular entity is performing. For example, you might have created an alert condition that opens an incident if a host's CPU goes above 80%. An uncovered entity is part of your system that is unmonitored which means there could be unhealthy behavior that goes unchecked. 
+
+We created [alert coverage gaps](https://one.newrelic.com/nrai/detection-gaps/home) to highlight those areas of your stack where you might be missing valuable data so you can prevent incidents before they happen. 
+
+## Find your coverage gaps [#find-your-coverage-gaps]
+
+On the Alerts and AI capabilites page, select alert coverage gaps in the naviagtion. There you will see a dashboard of all your entitites that are currently uncovered. 
```

Comment:
i think in general we can avoid 'capabilities' as a way to reference features/pages on the platform. reason being that it conflicts with our use of 'capabilities' in user mgmt space. So I'd just say 'On the **Alerts and AI** UI page...'

Reply:
thanks!